packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -426,96 +398,35 @@ export default class SalesforceCRMService implements CRM {
         records = results.records as ContactRecord[];
       }
 
+      if (recordToSearch === SalesforceRecordEnum.ACCOUNT && forRoundRobinSkip && !results.records.length) {
+        const attendeeEmail = emailArray[0];
+        // If we can't find the exact contact, then we need to search for an account where the contacts share the same email domain
+        const account = await this.getAccountBasedOnEmailDomainOfContacts(attendeeEmail);
+        if (account) {
+          records = [account];
+        }
+      }
+
       if (!records.length) {
         log.info("No records found");
         return [];
       }
 
-      // Handle owner information
```

Comment:
Huge win is removing the need to make a separate query for the owner information